src/main/resources/application.yml

```diff
@@ -47,6 +47,10 @@ spring:
           hmpps-auth:
             token-uri: ${oauth.endpoint.url}/oauth/token
 
+  mvc:
+    pathmatch:
```

Comment:
Required to get swagger docs out

Reply:
hmm.. not great that they change something that then requires magic config to work?! Again - might all "just work" for SpringDoc

Reply:
Added comment - and  raised another ticket to replace SpringFox with SpringDoc